src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx

```diff
@@ -19,18 +19,32 @@ const AspectRatioBox = styled(Box)<{ ratio?: ResponsiveValue<number> }>`
   ${ratioPadding}
 `
 
+const Container = styled(Box)<FontSizeProps>`
+  overflow: hidden;
+  ${fontSize}
```

Comment:
Oh woops I can remove this now; I was messing around with another way of doing this using font-size and em units but just using viewport units is easier to understand.